assets/benchmarks/shaders/Benchmark_VsSkyBox.hlsl

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+struct SceneData
+{
+    float4x4 ModelMatrix;                // Transforms object space to world space.
+    float4x4 CameraViewProjectionMatrix; // Camera's view projection matrix.
```

Comment:
is there a reason why you want to avoid passing local to projection matrix directly? (do the multiplication on the cpu side not in VS)